### PR TITLE
Remove Atomic Host as it is no longer supported upstream

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]

File: guide/latest/feature-sosreport.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 <div class="titlepage"><div><div><h2 class="title">
 <a name="feature-sosreport"></a>SOS Report</h2></div></div></div>
 <p>If present on the system Cockpit can use <code class="code">sosreport</code> to collect system configuration and diagnostic information.</p>
-<p>If <code class="code">sosreport</code> is not found on an Atomic system, cockpit will attempt to run <code class="code">sosreport</code> in the <code class="code">rhel7/support-tools</code> docker container.</p>
+<p>If <code class="code">sosreport</code> is not found on a system, cockpit will attempt to run <code class="code">sosreport</code> in the <code class="code">rhel7/support-tools</code> docker container.</p>
```
Comment (martinpitt) [inline guide/latest/feature-sosreport.html:28]: This bit needs to be dropped from this PR, as it gets auto-synced from cockpit: https://github.com/cockpit-project/cockpit/blob/master/doc/guide/feature-sosreport.xml I'm happy to do that myself tomorrow.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks for the cleanup!

**Comment by dustymabe** [comment]
Thanks @travier

**Comment by martinpitt** [comment]
I took the liberty to force-push your branch with reverting the guide/ bits. Thanks!

**Review by martinpitt — APPROVED** [review]

**Comment by martinpitt** [comment]
I sent https://github.com/cockpit-project/cockpit/pull/15702 to clean up the guide (and some related places).

**Pull request merged**